docs/tutorials/add-kaggle-credentials.md

```diff
@@ -0,0 +1,58 @@
+# Add Kaggle credentials to your FL sandbox
+
+Your Azure ML workspace has an attached key vault that can be used to store workspace-level secrets (users of the workspace will have access to it). We can use this to store Kaggle API key so that jobs can download data from Kaggle.
+
+This tutorial shows you how to add your Kaggle credentials to your FL sandbox.
+
+### Locate your workspace attached key vault
+
+You first need to locate your workspace key vault. It is provisioned by default in our [sandboxes](../provisioning/sandboxes.md) and is named `ws-shkv-<demoBaseName>`. You can find the name of your workspace in the Azure portal.
+
+### Option 1: using Azure CLI
+
+1. Let's first obtain your AAD identifier (object id) by running the following command. We'll use it in the next step.
+
+    ```bash
+    az ad signed-in-user show --query id
+    ```
+
+2. Create a new key vault policy for yourself, and grant permissions to list, set & delete secrets.
+
+    ```bash
+    az keyvault set-policy -n <key-vault-name> --secret-permissions list set delete --object-id <object-id>
+    ```
+
+    > Note: The AML workspace you created with the aforementioned script contains the name of the key vault. Default is `ws-shkv-fldemo`.
+
+3. With your newly created permissions, you can now create a secret to store the `kaggleusername`.
+
+    ```bash
+    az keyvault secret set --name kaggleusername --vault-name <key-vault-name> --value <kaggle-username>
+    ```
+
+    > Make sure to provide your *Kaggle Username*.
+
+4. Create a secret to store the `kagglekey`.
+
+    ```bash
+    az keyvault secret set --name kagglekey --vault-name <key-vault-name> --value <kaggle-api-token>
+    ```
+
+    > Make sure to provide the *[Kaggle API Token]((<https://github.com/Kaggle/kaggle-api#api-credentials>))*.
```

Comment:
Not sure why, but this link is not clickable - maybe because of the double parentheses or the <>?

I also prefer [this link](https://www.kaggle.com/docs/api#authentication) because it does not talk about environment variables etc, but still has the crucial information about where to click to get the token. (fairly arbitrary, decision is up to you)

Reply:
fixed!